src/redux/actions/projectActions.ts

```diff
@@ -88,6 +90,18 @@ export function saveProject(project: IProject)
     };
 }
 
+export function updateProjectTagsFromFiles(project: IProject)
+    : (dispatch: Dispatch, getState: () => IApplicationState) => Promise<void> {
+    return async (dispatch: Dispatch, getState: () => IApplicationState) => {
+        const projectCopy = Object.assign({}, project);
+        const projectService = new ProjectService();
+        const storageProvider = StorageProviderFactory.createFromConnection(project.sourceConnection);
+        await projectService.getTagsFromPreExistingFieldFile(projectCopy, storageProvider);
+        await projectService.getTagsFromPreExistingLabelFiles(projectCopy, storageProvider);
```

Comment:
Do we still need label files?

Reply:
I modified getTagsFromPreExistingLabelFiles to only add project tags that are missing from the label file. Before, it was overriding the current project tags from the labels.

Reply:
Sound good.